doc/user/content/ops/speedup.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Speedup"
```

Comment:
Do you mean like this or everything inside the same page?

<img width="325" alt="Screen Shot 2022-07-20 at 16 08 56" src="https://user-images.githubusercontent.com/11491779/180003267-9437c5a7-5b18-44e9-9a52-070998590fe1.png">

Reply:
With one of those titles, it could go in the same page. I imagine this page, for example, starting with a section on how to interpret an `EXPLAIN` plan.

Reply:
`EXPLAIN` is the best way to start the section but `EXPLAIN PHYSICAL` is the only one that displays something related to arrangements/indexes but is still super hard to interpret, so I had to leave it out as a comment 😞

After https://github.com/MaterializeInc/materialize/pull/12597 gets merged, it will be ten times better.

Reply:
Yes, sorry for the missing context! I meant it as something to add in the future, if we make the page more generic!